.github/workflows/stripe-patrons-data.yml

```diff
@@ -57,6 +57,50 @@ jobs:
             ~/.coursier
           key: sbt
 
-      - name: Build and upload to RiffRaff
+      - name: Build
         run: |
-          sbt "project stripe-patrons-data" riffRaffUpload
+          sbt "project stripe-patrons-data" "test; assembly"
```

Comment:
just check that if the tests fail, the assembly etc doesn't continue regardless

Reply:
Good point.
I just checked locally by breaking a test, and it does _not_ continue with the `assembly` step